.github/workflows/run.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12.0-alpha.6", "pypy3.9"]
```

Comment:
Regarding "3.12.0-alpha.6" and "pypy3.9", I checked the source code of @action/setup-python and tested it in my workflow.